packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -164,6 +164,7 @@ function AuthOptions({
 }: AuthOptionsProps): ReactElement {
   const { displayToast } = useToastNotification();
   const { syncSettings } = useSettingsContext();
+  const { trackSignup } = useContext(PixelsContext);
```

Comment:
Create a hook like the others so we can just usePixelsContext

Reply:
Done, even though I don't understand this practice 😬

Reply:
I think it is just to lessen the import, and quite easier to type one import than wait for the auto import to trigger twice while writing the code.